science/seqan-apps/Portfile

```diff
@@ -22,7 +20,11 @@ long_description    SeqAn applications for the analysis of large sets of sequenc
 
 homepage            https://www.seqan.de/applications/
 
-depends_build-append    port:boost port:coinor-liblemon
+depends_build-append \
```

Comment:
Ok, I have double checked and there seems be at least one app that links against boost libraries and not just the headers. So I think boost should be a also a lib dependency.

Reply:
The pull request should now have boost as a library dependency.